src/api/java/cvc5/Stat.java

```diff
@@ -0,0 +1,169 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Aina Niemetz, Andrew Reynolds, Abdalrhman Mohamed, Mudathir Mohamed
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * The cvc5 java API.
+ */
+
+package cvc5;
+
+import java.util.Map;
+
+/**
+ * Represents a snapshot of a single statistic value.
+ * A value can be of type `int64_t`, `double`, `std::string` or a histogram
+ * (`std::map<std::string, uint64_t>`).
```

Comment:
I'm not sure how you've handled this so far, but I think we should have the comment in terms of java types, not cpp types.

Reply:
Fixed. Let us say, the cpp documentation did not account for copying to Java :)